modules/reindex/src/test/java/org/elasticsearch/index/reindex/CancelTests.java

```diff
@@ -93,7 +92,7 @@ private void testCancel(String action, AbstractBulkByScrollRequestBuilder<?, ?>
 
         logger.debug("setting up [{}] docs", numDocs);
         indexRandom(true, false, true, IntStream.range(0, numDocs)
-                .mapToObj(i -> client().prepareIndex(INDEX, TYPE, String.valueOf(i)).setSource("n", i))
+                .mapToObj(i -> client().prepareIndex(INDEX, "_doc", String.valueOf(i)).setSource("n", i))
```

Comment:
I assume this is done because we don't yet have a typeless indexing method?

Reply:
Correct